css/style.css

```diff
@@ -801,10 +801,20 @@ div#content
 	background-color: #f6f6f6;
 	font-size: 0.875em;
 	line-height: 1.4em;
-	text-align: justify;
 	border: 0.25em solid #ccd5d5;
 }
 
+/*  This is a CSS hack to only target Blink based browsers
+    Targets Chrome 28 and later and Opera 14 and later
+*/
+@media all and (-webkit-min-device-pixel-ratio:0) and (min-resolution: .001dpcm) {
```

Comment:
I wanted to avoid JS if at all possible because I don't like putting style inside of anything but the CSS, as it has a high probability of being forgotten and left in there for a while. And I figured that it wasn't so bad as this will probably be removed soon when Chrome supports hyphens.

If you still want me to change this to a JS solution even with the above problems, please let me know.